pom.xml

```diff
@@ -33,12 +33,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <junit5.version>5.10.2</junit5.version>
-        <assertj.version>1.7.25</assertj.version>
-        <jackson.version>2.17.1</jackson.version>
-        <commons-lang3.version>3.14.0</commons-lang3.version>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
```

Comment:
This is fine to remove because we have `<release>11</release>` in the `maven-compiler-plugin` configuration.